app/src/main/java/com/example/grutor/Activites/RegisterActivity.java

```diff
@@ -25,10 +25,14 @@ public class RegisterActivity extends AppCompatActivity {
 
     public  EditText etUserLogin;
     public  EditText etPassword;
+    public EditText etFirstName;
+    public EditText etLastName;
+    public String userFullName;
+
     public Button btnNext;
     final Fragment fragmentRegister = new RegisterFragment();
     public static final String TAG = "RegisterActivity";
-    protected ParseUser user;
+    public static ParseUser user;
```

Comment:
It is not a good way to define a static and pass to other here. you can pass the username and password to fragmentRegister directly, then signup a user.